vignettes/monaLisa.Rmd

```diff
@@ -361,41 +360,45 @@ Only distal autosomal peaks (distance > 1kb from any TSS) were kept since promot
 
 -->
 
-## Load Data
-
-In our example dataset, we have quantified ATAC-seq reads on enhancers in mouse P0 lung and liver tissues. The log2-fold change (our response vector in this example) is for liver vs lung chromatin accessibility. We are using a set of 10,000 randomly sampled enhancers to illustrate how randomized lasso stability selection can be used to select TFs.
+## Load library
```

Comment:
Maybe better `Load packages` instead of `Load library` :-) (there is a long ongoing discussion about the wrong use of libraries to refer to R packages: https://www.r-bloggers.com/2013/01/packages-v-libraries-in-r/)

Reply:
:D